tests/generator/test_rom.py

```diff
@@ -16,6 +16,7 @@
 from chia.wallet.puzzles.load_clvm import load_clvm
 
 MAX_COST = int(1e15)
+COST_PER_BYTE = int(0)
```

Comment:
I took the easy way out here by setting this to 0, to preserve all the existing tests

Reply:
If it's not too much trouble, can you add a second test which sets `COST_PER_BYTE` to 1, with a different `EXPECTED_COST`? Might be handy.

Reply:
testing this, it turns out none of the costs this test look at depend on the cost per byte. The cost per byte in `get_name_puzzle_conditions()` is only used to limit the max cost. As long as the program completes with a lot of margin, the cost per byte won't matter.